### ISSUE TITLE
t3kit_composer dependencies no longer support t3kit master branch (TYPO3 version 8).

### ISSUE THREAD

**Issue by Nguonchhay** [t3kit/t3kit_composer#24]
I got this error while running command `./initWithComposer.sh`

```

Loading composer repositories with package information
Updating dependencies (including require-dev)                                 
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for typo3-ter/theme-t3kit 1.10.1 -> satisfiable by typo3-ter/theme-t3kit[1.10.1].
    - typo3-ter/gridelements 7.0.5 requires typo3/cms-core >= 7.6.2, <= 7.99.99 -> satisfiable by typo3/cms[7.6.10, 7.6.11, 7.6.12, 7.6.13, 7.6.14, 7.6.15, 7.6.16, 7.6.2, 7.6.3, 7.6.4, 7.6.5, 7.6.6, 7.6.7, 7.6.8, 7.6.9].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.10].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.11].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.12].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.13].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.14].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.15].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.16].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.2].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.3].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.4].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.5].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.6].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.7].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.8].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.9].
    - t3kit/theme-t3kit dev-master requires typo3/cms ^8.5 -> satisfiable by typo3/cms[8.x-dev].
    - typo3-ter/theme-t3kit 1.10.1 requires typo3-ter/gridelements >= 7.0.5, <= 7.0.99 -> satisfiable by typo3-ter/gridelements[7.0.5].
    - Installation request for t3kit/theme-t3kit dev-master -> satisfiable by t3kit/theme-t3kit[dev-master].

```

**Comment by sakonachhoeurng** [comment]
I also got this problem too.

**Comment by franzkugelmann** [comment]
Me too. I also tried to run `composer update --no-dev` and `composer update --no-dev --prefer-stable` but this does not help, because as discussed in https://github.com/composer/composer/issues/5652 it is not supported to require the same package in "require" and in "require-dev", as opposed to what the documentation seems to say.
If you remove all duplicate packages from require-dev (in main composer.json), then it works.

**Comment by edekowalski** [comment]
Me too ..

  Problem 1
    - Installation request for typo3-ter/theme-t3kit 1.10.1 -> satisfiable by typo3-ter/theme-t3kit[1.10.1].
    - typo3-ter/gridelements 7.0.5 requires typo3/cms-core >= 7.6.2, <= 7.99.99 -> satisfiable by typo3/cms[7.6.10, 7.6.11, 7.6.12, 7.6.13, 7.6     .14, 7.6.15, 7.6.16, 7.6.2, 7.6.3, 7.6.4, 7.6.5, 7.6.6, 7.6.7, 7.6.8, 7.6.9].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.10].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.11].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.12].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.13].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.14].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.15].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.16].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.2].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.3].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.4].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.5].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.6].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.7].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.8].
    - Can only install one of: typo3/cms[8.x-dev, 7.6.9].
    - t3kit/theme-t3kit dev-master requires typo3/cms ^8.5 -> satisfiable by typo3/cms[8.x-dev].
    - typo3-ter/theme-t3kit 1.10.1 requires typo3-ter/gridelements >= 7.0.5, <= 7.0.99 -> satisfiable by typo3-ter/gridelements[7.0.5].
    - Installation request for t3kit/theme-t3kit dev-master -> satisfiable by t3kit/theme-t3kit[dev-master].

Best Regard's and thx 
Ede

**Comment by dmh** [comment]
fixed in version 0.2.2
